src/detect-http-headers-stub.h

```diff
@@ -145,26 +141,31 @@ static void DetectHttpHeadersRegisterStub(void)
     sigmatch_table[KEYWORD_ID].url = "/rules/" KEYWORD_DOC;
     sigmatch_table[KEYWORD_ID].Setup = DetectHttpHeadersSetupSticky;
     sigmatch_table[KEYWORD_ID].flags |= SIGMATCH_NOOPT | SIGMATCH_INFO_STICKY_BUFFER;
+#ifdef KEYWORD_ID_LEGACY
```

Comment:
expanding the size of this table is not free, it will lead to more mem use. Do we only have this here for supporting commandline output options? If so I rather see it be hard coded to that.